openstack/identity/v2/tokens/requests.go

```diff
@@ -53,7 +54,7 @@ type AuthOptions struct {
 }
 
 // ToTokenV2CreateMap builds a token request body from the given AuthOptions.
-func (opts AuthOptions) ToTokenV2CreateMap() (map[string]interface{}, error) {
+func (opts *AuthOptions) ToTokenV2CreateMap() (map[string]interface{}, error) {
```

Comment:
Why have you changed this method to have a pointer receiver?

Reply:
I thought it was required to allow it to take a structure that implements the `AuthOptions` interface? Is that wrong?

Reply:
You're correct.